package.json

```diff
@@ -16,10 +16,11 @@
         "pretty": "prettier --check scripts/**/*.js",
         "prettify": "prettier --write scripts/**/*.js",
         "lint-md": "remark -q -f content",
+        "lint-js": "eslint .",
         "build-json": "node scripts/build-json/build-json.js",
         "scrape-mdn": "node scripts/scraper/scrape-mdn.js",
         "spell-md": "mdspell -a -n -r -x --en-us 'content/**/!(*contributors).md'",
-        "test": "npm run lint-md && npm run spell-md"
+        "test": ""
```

Comment:
Why this? 
Perhaps `npm run test` could be a lazy shortcut to run everything but in `.travis.yml` we're being more explicit.

Reply:
> I'd rather remove this entirely and put this in .travis.yml instead

Sorry maybe i misunderstood.

I agree it would be beneficial to `npm run test` locally too and it should go in both places.

Reply:
Some day, when we can try GitHub actions we might want to try to set up a matrix so that JS linting and content linting (i.e. markdown and spelling) is done in two parallel tracks. 

By having them broken out as we currently do (in this here PR), in travis, it means that all of them get a chance to run. It's the equivalent of `npm run lint-js ; npm run lint-md ; npm run spell-md`. 
However, please re-instate the "test" one as `npm run lint-js && npm run lint-md && npm run spell-md`.

Reply:
Sorry for the misunderstanding 😄, should be better now.